postgres-client-builder/Dockerfile

```diff
@@ -30,21 +30,16 @@ RUN wget https://www.postgresql.org/media/keys/ACCC4CF8.asc && \
     apt-key add ACCC4CF8.asc && \
     echo "deb-src https://apt-archive.postgresql.org/pub/repos/apt focal-pgdg-archive main" >> /etc/apt/sources.list.d/pgdg.list && \
     apt-get update && \
-    apt-get source postgresql-10=10.16-1.pgdg20.04+1
+    apt-get source postgresql-15=15.3-1.pgdg20.04+1
 
 ## Compile postgres client
-#
-# `make -C postgresql-$PG_VERSION/src/backend` is required to ensure that the
-# header files are properly generated for Postgres 10. This is not required
-# to build the client tools in v11+.
 RUN cd postgresql-$PG_VERSION && \
     env CC=gcc CPPFLAGS=-I/usr/local/ssl/include LDFLAGS=-L/usr/local/ssl/lib ./configure --with-openssl --without-readline --prefix=/usr/local/pgsql  && \
```

Comment:
Minor comment, according to [postgres documentation](https://www.postgresql.org/docs/current/install-procedure.html#CONFIGURE-OPTIONS), --with-openssl is deprecated in favours of --with-ssl=openssl